### DISCUSSION TITLE
BlackHole device not showing up in Audio MIDI setup on MacBook M1 Big Sur

### DISCUSSION THREAD

**Discussion by jguigui** [ExistentialAudio/BlackHole#293] [Q&A]
Hi,
I've tried to install **BlackHole 0.2.9**, both the 2-channels and 16-channels version, on a MacBook Pro 13'', running with Apple Silicon **M1** and **Big Sur**.
The installation process is completing properly. It ends with a message "installation successful".
But the BlackHole interface is **not visible in the Audio MIDI Setup** list of devices. I have restarted the system, no effect.
I tried then with Loopback from RogueAmoeba, and Loopback appeared in the list of devices. But not BlackHole.
Anyone experimenting the same issue?
Thanks in advance,
Jean-Paul

**Comment by devinroth** [comment]
Test out the beta version on the m Apple Silicon thread.

Devin
